readme.md

```diff
@@ -197,6 +197,8 @@
 - [Statusfy](https://github.com/bazzite/statusfy) - Status page system using Tailwind CSS.
 - [Todolist](https://github.com/guillaumebriday/todolist-frontend-vuejs) - To-do list application using Tailwind CSS.
 - [LeagueStats](https://github.com/vkaelin/LeagueStats) - Statistics website for League of Legends players.
+- [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - ecommerce storefront using svelte & Tailwind CSS.
```

Comment:
```suggestion
+ [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - E-commerce storefront using Svelte & Tailwind CSS.
```

Reply:
Can you update this to respect capitalization rules? After this change I think we can merge. :)